dnf/cli/format.py

```diff
@@ -15,8 +15,23 @@
 # Red Hat, Inc.
 
 from __future__ import unicode_literals
+from datetime import datetime
```

Comment:
I this case, I think it is ok  to use from x import y, python is kind of silly here, with the datetime.datetime and a datetime.datetime prefix, is not very useful